src/adapters/compat.js

```diff
@@ -25,6 +25,10 @@ export default class DatabaseAdapterCompat {
     return this.underlyingAdapter.schema
   }
 
+  get dbName(): ?string {
+    return this.underlyingAdapter._dbName
```

Comment:
I'd prefer if the underlying adapters also had a `dbName`, not a `_dbName`. the compat layer is meant to turn callback-based functions into promise-returning functions.

Reply:
I have renamed _dbName into dbName (see https://github.com/Nozbe/WatermelonDB/pull/1227/commits/cab09a2d87a11c872b729db28c59f8f90d050424)